zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -185,9 +180,9 @@ object Response {
   /**
    * Creates a response with content-type set to application/json
    */
-  def json(data: String): Response =
+  def json(data: CharSequence): Response =
```

Comment:
We can do this for `Response.text("abc")` also.

Reply:
I'll add